server/src/query/popularity.ts

```diff
@@ -0,0 +1,23 @@
+import { Prisma } from "@prisma/client";
+import { prisma } from "../model";
+
+export async function recordContentView(
```

Comment:
docstring

Reply:
Hah, all you did was move this function to a different file, and you get saddled with making the docstring!